### ISSUE TITLE
Glasklart Open Thread 2016

### ISSUE THREAD

**Issue by dreamnet** [glasklart/hd#7000]
Because #1323 takes waaaayyyy to long to load on slower network connections, I'm starting a new open thread. I let this open until 2017.

**This thread ist the place where everyone can just shoot the crap, ask random stuff, etc.**

Happy discussing! :wink: 

/cc @glasklart/maintainers @glasklart/feature-contributors

**Comment by Trubbel** [comment]
@dreamnet I'm getting a bunch of errors on the [Glasklart iTunes Data Crawler](http://getart.dreamnet.at/) and the icon preview's doesn't seem to work.

**Comment by dreamnet** [comment]
@Trubbel Ah yes, I see... maybe my hosting provider has changed some server parameters... I check it in a view hours (I'm at work at the moment).

**Comment by Anuskuss** [comment]
When will the new version get released?

**Comment by dreamnet** [comment]
@Anuskuss This weekend.

**Comment by dreamnet** [comment]
Made some [Glasklart mock-ups](http://glasklarthd.dreamnet.at/screenshots) if someone is interested.

**Comment by dreamnet** [comment]
@glasklart/maintainers Today, my webserver was hacked. :angry:  I had to take down glasklarthd.dreamnet.at, the app data crawler and the preview script. I am working to get them up again, :sweat: but at the moment I cannot say how log it takes. Sorry for this. Be patient. :zzz:

**Comment by Trubbel** [comment]
Oh my god, some people just have to ruin things for everyone else. :angry: 

Do you know how they manage to do it? So you can prevent it from happening again?

**Comment by dreamnet** [comment]
@Trubbel Yeah, some people seem to have nothing else to do.... :rage: 
Currently we - my webspace provider an I - don't know how they did it, but it seems that they have inserted some code - actually it was just advertising for viagra and other generic pills - in my scripts. Now I have to check all the scripts and the databases for "strange" code and if I'm shure it's clean I have to change all server passwords bevor going online again. This may take a view days, because there are hundreds of scripts to check. :cold_sweat: So as I mentioned above - be patient (and do some icons in the meanwhile :wink:). I'm working.

**Comment by dreamnet** [comment]
@glasklart/maintainers The preview script and the data crawler are back up again. :smile:

**Comment by ivanov007** [comment]
wow, that was quick, and dam at least we know the theme is popular enough to make hackers look this way :sweat:

**Comment by dreamnet** [comment]
Uhhh... I was too optimistic - I'm already working on the crawler....

**Comment by dreamnet** [comment]
...finished. :cold_sweat:

**Comment by Trubbel** [comment]
@dreamnet Awesome work as always! :beers: Thanks for all the hard work as well.

I hope this is the last time though, they better do something useful with their time instead of ruining it for others. :disappointed:

**Comment by dreamnet** [comment]
@glasklart/maintainers OK, after three days solving a lot of problems with the webserver and the domain itself, http://glasklarthd.dreamnet.at, http://getart.dreamnet.at and http://preview.dreamnet.at are up again - and stronger than before. :stuck_out_tongue_winking_eye:
Sorry for the downtime! I have to re-activate a bunch of other websites on my server, so don't expect me to work on some icons for the next two or three days.

**Comment by Anuskuss** [comment]
I have a theming question. Is it possible to give some specific icons a overlay? I want to add a little controller symbol in the bottom left corner of the Glasklart icons, so I can tell if they have MFI controller support or not. But I don't want to overwrite the Glasklart.theme and want to avoid having duplicate icons.

Is there a simple way to achive this?

**Comment by dreamnet** [comment]
@Anuskuss No, I think there is no way than to change the icon itself.

**Comment by Anuskuss** [comment]
@dreamnet So the best bet would be to create a seperate .theme with all the icons I want and then place it higher in Anemone?

**Comment by dreamnet** [comment]
@Anuskuss Yes, I think this would be the best way.

**Comment by nekholm** [comment]
[This is such a great feeling](http://www.livememe.com/fpcsfgh.jpg)

**Comment by Trubbel** [comment]
@nekholm Agreed. :+1: We try our best as well. :relaxed:

**Comment by Anuskuss** [comment]
Do you plan to create a @3x version for 5.0?

**Comment by dreamnet** [comment]
@Anuskuss Yes, I'm planing to make all Glasklart icons 180x180px and use Anemone's -large method for 4.0. But this depends on the release of Anemone 1.2.

**Comment by Anuskuss** [comment]
Would it be a seperate theme since it would be unnecessary bloat for the @2x devices (and for the @3x devices to include a @2x version for that matter)?

**Comment by dreamnet** [comment]
@Anuskuss We had a dicussion about this on #6595. Removing the background (baseplate) from the Glasklart icons should save about 20MB space so we would end up with about the same size Glasklart has actually.

**Comment by coolstar** [comment]
Just a heads up, Anemone 1.1-163 on the fast ring may be marked as stable in a few days. Please try it out and let me know how it works out for all of you :)

**Comment by Anuskuss** [comment]
What should we do about the calender icon in tweaks which uses AppList?
![](https://cloud.githubusercontent.com/assets/7157389/13161918/50c24c5a-d6a1-11e5-9331-933957d8e416.PNG)

Is this a Glasklart, an Anemone or an AppList problem?

**Comment by dreamnet** [comment]
@Anuskuss Hm, hard to say - normally, Anemone should take this icon: `Bundles/com.apple.preferences-ui-framework/Calendar@2x.png` (on iOS9). But Anemone seems to take `IconBundles/com.apple.mobilecal@2x.png`, with is logically just a blank baseplate because it is used on springboard to display the actual date. I have no idea how to change this...

**Comment by Anuskuss** [comment]
Just noticed that it also happens on Notification Center so it's your turn @coolstar :wink: 
![](https://cloud.githubusercontent.com/assets/7157389/13199917/756ce6f2-d834-11e5-8e74-3ffc24410354.PNG)

**Comment by nekholm** [comment]
I don't think it's coolstar's fault, this already happened before the switch to Anemone. I know the icon was blank when using LockInfo, and it has been that way for quite a long time.

**Comment by coolstar** [comment]
I'll take a look at it.


Also @dreamnet, @Trubbel; if I could get your email addresses, I can add you to the Anemone slack (chat room). We have the other Anemone developers and many others from the theming community in that discussion :)

**Comment by Trubbel** [comment]
@coolstar You can find mine on my profile. :wink:

**Comment by dreamnet** [comment]
Anybody knows what's wrong here on Github, a view hours ago I fulfilled a lot of open requests and now they are ... simply gone. Example is #7297. ??? :confused: :question:

**Comment by dreamnet** [comment]
@Trubbel Ah, now I see, @Trubbel is not a human... :laughing:. Anyway, all Issues are back now. All fine.

**Comment by Trubbel** [comment]
@dreamnet Yeah the Github staff just unflagged my account and made it public again. :laughing:
There shouldn't be any issues now haha..

**Comment by Anuskuss** [comment]
Could we get another compass icon? I don't like it tbh.

**Comment by coolstar** [comment]
@Trubbel @dreamnet btw. check your mails. I sent you an invite to the Anemone slack :)

**Comment by coolstar** [comment]
@Trubbel @dreamnet just checking if you got your invites as I haven't seen you both on the Anemone slack yet :P

**Comment by Trubbel** [comment]
@coolstar Pretty sure Hotmail ate the mail then, can't find it in junks either.. But I guess that's what I get for still using Hotmail after all these years..

**Comment by coolstar** [comment]
@Trubbel alright I've re-sent the invitation so you should get it now

**Comment by dreamnet** [comment]
@coolstar Got the invitation, but I have a full time job and I'm very busy in my free time these days, so I dont know if I can find the time join the discussion. :clock1030: :cry:

**Comment by boxermansr** [comment]
Hi; hoping someone can help me out. Since appinfo no longer works, and I don't have any of the other tools to get APPID or whatever, I am hoping someone would be so kind as to submit/make a request for a Cydia app for me, please ;) - its called CellularInfo and its on the repo nahum365.github.io . I know it's a big ask, but i'm hopeful someone else out there is using the app/utility (shows what band LTE you're connected to, good if the field test app on the iPhone isn't working right, as is my case).

Thanks a **ton**

**Comment by Trubbel** [comment]
@boxermansr You should be able to use Appster to find out applications Bundle IDs. But I did it for you.

**Comment by boxermansr** [comment]
@Trubbel woops, I completely forgot about Appster, sorry. I was away from having a jail broken device for a few months, and just got one that I was able to jail break, getting all my stuff back on there, and Appster was one of the biggies. Thanks for reminding me of that, and also for submitting the request, I appreciate it.

**Comment by Trubbel** [comment]
@boxermansr No problem. :+1:

**Comment by nekholm** [comment]
Too bad Appster doesn't show iTunes IDs, or am I missing something? For Cydia apps it works.

**Comment by dreamnet** [comment]
@nekholm To find out the iTunes ID you can use iTunes - we have a [https://github.com/glasklart/hd/wiki/How-to-Submit-an-Icon-Request#step-1-of-2-find-the-itunes-id](WIKI entry) for this. You can also use the 'App Store' box on my website for searching - if you find the app and open it in your web browser, you will find the iTunes ID in the link.

**Comment by ivanov007** [comment]
@nekholm you need to tap on export and you will get the iTunes ID

**Comment by Trubbel** [comment]
@dreamnet I'm not sure if I'm the only one, but the previews doesn't seem to be working?
Can anyone confirm?

**Comment by nekholm** [comment]
@Trubbel @dreamnet I noticed this too. The original image shows, but not the others.

**Comment by ivanov007** [comment]
same here

**Comment by Trubbel** [comment]
Everything seems to be in order now.

**Comment by dreamnet** [comment]
Sorry, I was away last week... all fine again?

**Comment by dreamnet** [comment]
:star2: **After adding some icons today, Glasklart has now over 9.000 icons!** :star2:
Next release will be this weekend.

**Comment by Anuskuss** [comment]
I just want to thank @dreamnet for taking his free time to fulfil my requests. Danke man, you're the best!

**Comment by dreamnet** [comment]
@Anuskuss :+1: :sweat_smile:

**Comment by grkka** [comment]
Thanks a lot! Jailbreak again in 9.3.3 and again I already have Glasklart! @dreamnet @trubbel @ivanov007

**Comment by dreamnet** [comment]
I'm off for about seven days. Just relaxing and climbing on a view hills... :palm_tree:

**Comment by ivanov007** [comment]
@dreamnet great, have a nice week off

**Comment by jfelchner** [comment]
I'm so glad to see that you all are still hard at work on this. 😀 

I can't believe @dreamnet is still hitting it hard! :+1:

Max had no idea his little theme was going to get this popular hahahaha  Almost 10,000 icons!!!  7,600 issues!!!

CRAZY!!

Keep it up!!

**Comment by dreamnet** [comment]
@jfelchner Hey Jeff, glad to her from you! :+1: 
And yes, still here, still working for Glasklart, still love the theme. :heart: 
Yesterday, I was working 30 years for the same company (yes, I am THAT old :wink:), I think I will do this also for Glasklart. :laughing: 
P.S.: Na, not 10.000 icons - about 5.500 in the next release, because we are going up to 180x180px and for this we have to throw away a lot of older icons.... but that's also a cleanup.

**Comment by ivanov007** [comment]
have you wonder how glasklart will look on iOS 10?
http://www.idownloadblog.com/2016/09/15/ios-10-jailbreak-hacker-luca-tedesco-shows-off-ipad-with-mobilesubstrate-support/

**Comment by dreamnet** [comment]
@ivanov007 Woooo! Great news! 🎆

**Comment by jfelchner** [comment]
Oh my gosh that's so cool!!!

**Comment by BrokenCommander** [comment]
Hey guys :v:

I recently discovered Glasklart and I must say I like the simplicity of the theme - it is minimalist, clean and is still being updated. Almost everything on my device is themed but I certainly don't use a lot of apps anyway.
I want to contribute some of my missing apps and I have a question:

Is it necessary to look *exactly* like the original app? If it's okay I would like to make some apps even cleaner than they already are. For example:

<img src="https://camo.githubusercontent.com/3cf9234b2105acd28e74e3d9354c15e1a12b5913/687474703a2f2f6973312e6d7a7374617469632e636f6d2f696d6167652f7468756d622f507572706c65342f76342f63302f30622f65322f63303062653263312d356431662d376634622d356432662d3238386633326636633261392f6d7a6c2e6c6f7a6a67716b6e2e706e672f30783073732d38352e6a7067" width=200> <img src="https://camo.githubusercontent.com/d1f94eda42950244f6e9290f6428d95705e8dffe/687474703a2f2f707265766965772e647265616d6e65742e61743f696d6167653d68747470733a2f2f636c6f75642e67697468756275736572636f6e74656e742e636f6d2f6173736574732f323036383133302f393538353937322f35323736666462612d353031612d313165352d396330612d3435376632346130613864332e706e67" width=200> <img src="http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/7157389/18811045/34f75f90-82a6-11e6-8d3b-f224b24a689c.png" width=200>

<img src="https://camo.githubusercontent.com/47d51759c25caacd923eaa100f91e34ec6a41fba/68747470733a2f2f6973342d73736c2e6d7a7374617469632e636f6d2f696d6167652f7468756d622f507572706c6536322f76342f32332f66312f38342f32336631383432362d633034392d346565642d346662622d3934653930303137376131392f736f757263652f31303234783130323462622e706e67" width=200> <img src="https://camo.githubusercontent.com/98ebd3deabc8c490fb9700dd4d9a8c20a5e54ecc/687474703a2f2f707265766965772e647265616d6e65742e61743f696d6167653d68747470733a2f2f636c6f75642e67697468756275736572636f6e74656e742e636f6d2f6173736574732f323036383133302f31383737333931372f66646338316435612d383135332d313165362d396234332d3533636435623461346434382e706e67" width=200> <img src="http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/7157389/18811126/88edf284-82a7-11e6-9372-d30897d8ebef.png" width=200>

<img src="https://camo.githubusercontent.com/1e3ce43d91b63ac50170b59532eaba83391a3917/687474703a2f2f6973342e6d7a7374617469632e636f6d2f696d6167652f7468756d622f507572706c65352f76342f65322f63382f39382f65326338393863612d383939632d306131342d373237652d6230373363356233363163372f736f757263652f31303234783130323462622e706e67" width=200> <img src="https://camo.githubusercontent.com/b3ab7089bd7955cce262ccb4c6f7c40c8adc2417/687474703a2f2f707265766965772e647265616d6e65742e61742f3f696d6167653d68747470733a2f2f636c6f75642e67697468756275736572636f6e74656e742e636f6d2f6173736574732f31303733303132322f31323837343830372f65396361663433652d636464622d313165352d393865332d3662306164313435376633612e706e67" width=200> <img src="http://preview.dreamnet.at/?image=https://cloud.githubusercontent.com/assets/7157389/18812112/658deeb2-82c9-11e6-9878-7cf7e1752342.png" width=200>

**Comment by dreamnet** [comment]
@BrokenCommander It is not always necessary to go exactly with the original - for some icons it's also not possible, because we have only white and transparent (btw black and transparent in the source files). But the app must be recognizable. The icons you posted above are very good, so I would put it in the theme. If you want to contribute, you are cordially invited! 🎩

**Comment by nekholm** [comment]
Just wanted to say a big **thank you** to @2Qt2beStr8 and @dreamnet for quickly making new icons for those that went missing in Glasklart 5.00. Around 15 of my app icons needed to be remade, and they were all done within a day, probably less than half a day.

**Comment by dreamnet** [comment]
@nekholm Thanks 🎩, we do our best. We have made [over 250 commits](https://github.com/glasklart/hd/compare/release/v5.00.0...master) to the master since the release of 5.0. Other helpers: @Trubbel, @ivanov007, @l8er4now and @Checkit23. Also thanks guys!
5.01 will be submitted to the modmyi repo in ~ two hours.

**Comment by quantum8** [comment]
You beat me to it @nekholm! Big thanks to the contributers who have gone above and beyond in recreating icons  almost as fast as I've submitted the requests. You guys are fantastic!

**Comment by BrokenCommander** [comment]
Is it only me or is the preview broken?

**Comment by dreamnet** [comment]
@BrokenCommander Imagick seem to be gone on my webserver... I'm currently try to figure out why.

**Comment by dreamnet** [comment]
Can't resolve the problem with Imagick by myself, so I have opened a ticket by my provider.

**Comment by dreamnet** [comment]
Problem with Imagick solved, preview is working again.

**Comment by dreamnet** [comment]
I have to say
`THANK YOU!` 👏  
===
to @2Qt2beStr8 for fulfilling some of the older and hard to-do icon requests in the last weeks. I can't remember when we were down to 40 open issues, but it must be months ago. Keep on your good work!

**Comment by grkka** [comment]
Thank you @2Qt2beStr8 and thank you to everyone else

**Comment by Trubbel** [comment]
I agree, @2Qt2beStr8 you have done some really good work these couple of weeks. Cheers! :beers:

**Comment by 2Qt2beStr8** [comment]
i thank you all for the nice words  👍

**Comment by dreamnet** [comment]
@jfelchner Can you please add @2Qt2beStr8 to the team? He has done very good work in the last weeks, has a good eye and I think he will be a a great help.
THX!

**Comment by jfelchner** [comment]
@dreamnet absolutely.  Hey @2Qt2beStr8 why don't you add a profile icon ;)

**Comment by jfelchner** [comment]
@2Qt2beStr8 you should be getting an email shortly.

**Comment by jfelchner** [comment]
@dreamnet I also want to make sure that everyone currently on the maintainers list is still active.

Here's the full list:

* MagnaMike
* scottbuckley
* vinstic
* ivanov007
* Trubbel
* dreamnet

If they haven't been regularly active in the last year, let's remove them.  We can add them back later if necessary.

**Comment by dreamnet** [comment]
@jfelchner OK,
I think it's save to remove @MagnaMike, @scottbuckley and @vinstic. What's on with @domegranate?

**Comment by 2Qt2beStr8** [comment]
@jfelchner thank you for adding to the team :+1:

**Comment by Trubbel** [comment]
@2Qt2beStr8 Welcome aboard! :wink:

**Comment by jfelchner** [comment]
@dreamnet oh yeah. He's on there too. :)

**Comment by jfelchner** [comment]
@2Qt2beStr8 nice profile photo! 👍🏻

**Comment by jfelchner** [comment]
@dreamnet Glasklart Github teams are now all cleaned up.  Lemme know if you need anything else.

**Comment by MagnaMike** [comment]
@dreamnet @jfelchner, yes it was fine to remove me. I don't have iOS devices anymore and I just haven't had the time to do anything on it. Keep up the good work - two thumbs up! 👍 👍

**Comment by 2Qt2beStr8** [comment]
@dreamnet why you give all icons i have made to @Trubbel?

**Comment by 2Qt2beStr8** [comment]
@dreamnet ... stop!! what is wrong with my icons? i craete them and i also would have the credits!

**Comment by dreamnet** [comment]
@2Qt2beStr8 Wooops... sorry! 🙏Forgot to switch the Github user.... 😢

**Comment by 2Qt2beStr8** [comment]
that's not good .. i spent a lot of time to create this 30 icons :-(

**Comment by dreamnet** [comment]
@2Qt2beStr8 I'm counting 26. I correct it after clearing out all my unread messages...

**Comment by jfelchner** [comment]
I really wish @2Qt2beStr8 would stop updating icons. 😂  I get a push notification for _every single one._ 😳  If only Github had a way to unsubscribe from every issue in a repo. 🙄

**Comment by 2Qt2beStr8** [comment]
@jfelchner ... sorry Jeff ... should i stop for a few days?

**Comment by jfelchner** [comment]
@2Qt2beStr8 no it's ok :)  It's gonna happen some time.  I'm just unsubscribing from each one as I get the notification for it.

**Comment by jfelchner** [comment]
@2Qt2beStr8 how are you going through them and deciding which ones to do?  Like, what's your process?

**Comment by 2Qt2beStr8** [comment]
i use the "ToDo" section from @dreamnet 's homepage. there a so many icons which need to be updated. now i make the easier ones :-) - that is the reason why i can make so much

**Comment by jfelchner** [comment]
Only 2,978 left to go 😂  *sigh* ok :)

**Comment by 2Qt2beStr8** [comment]
yes, i know ... this is really very much

**Comment by ivanov007** [comment]
hey @jfelchner have you tried to disable the notifications from github settings?
![screenshot 16-10-24 09 19 09](https://cloud.githubusercontent.com/assets/3090247/19649385/39463da0-99cb-11e6-861b-012c108b1faa.png)

**Comment by jfelchner** [comment]
@ivanov007 thanks man, but unfortunately I do need notifications for other repos. ☹️

**Comment by 2Qt2beStr8** [comment]
@dreamnet and @jfelchner i have a problem. i recieved this message from github
One of our mostly harmless robots seems to think you are not a human.
Because of that, it’s hidden your profile from the public. If you really are human, please contact support to have your profile reinstated. We promise we won’t require DNA proof of your humanity.

**Comment by dreamnet** [comment]
@2Qt2beStr8 😄 @Trubbel had the [same problem](https://github.com/glasklart/hd/issues/7000#issuecomment-190743262). I'm sure he knows how to solve this.

**Comment by 2Qt2beStr8** [comment]
i wrote a message to the github support and the problem no longer exists :-)

**Comment by jfelchner** [comment]
@2Qt2beStr8 YOU'RE A MACHINE! 🤖 

😉

**Comment by 2Qt2beStr8** [comment]
Yes ... i am a Icon-creating-machine 😂

**Comment by dreamnet** [comment]
@jfelchner Take a look at [this](https://github.com/glasklart/hd/graphs/contributors)... @2Qt2beStr8 will beat you in not time... 😁

**Comment by Trubbel** [comment]
@2Qt2beStr8 Wow! You are on :fire: :grinning:

**Comment by jfelchner** [comment]
@dreamnet I'd love that! <3

**Comment by Trubbel** [comment]
Good news, seems like we will be getting a new jailbreak for 10.1.1 soon. So if you have any phones above that version I would recommend downgrading.

**Comment by Trubbel** [comment]
Good news, seems like we will be getting a new jailbreak for 10.1.1 soon.

**Comment by Trubbel** [comment]
@dreamnet Apple's News application has a [different artwork](http://getart.dreamnet.at/?id=1066498020) on iOS 10, should we open a new issue or stay with the old one?

**Comment by grkka** [comment]
@Trubbel You already have Glasklart with Jailbreak in iOS 10?

**Comment by Trubbel** [comment]
@grkka I haven't jailbroken yet. But the new jailbreak is out, but it is currently very unstable. So I would recommend waiting on using the program.

**Comment by dreamnet** [comment]
@Trubbel Because we are not supporting iOS 10 official, I think we should currently stay with the with the "old" icon.

**Comment by Trubbel** [comment]
@dreamnet Yeah that makes sense, fair enough. :smiley:

**Comment by dreamnet** [comment]
Merry Christmas and a happy new year ! :christmas_tree:

**Comment by 2Qt2beStr8** [comment]
Merry Xmas to all! 🌲

**Comment by Trubbel** [comment]
Merry Christmas :snowman_with_snow:

**Comment by jfelchner** [comment]
I missed the signing window for 10.1 by one day 😢

**Comment by grkka** [comment]
Merry Christmas to all!!!
